Lyra/Presentation/Scenes/IntroCutscene/View/CutsceneView.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  CutsceneGuideView.swift
+//  Lyra
+//
+//  Created by Luiz Sena on 11/04/23.
+//
+
+import SwiftUI
+
+struct CutsceneView: View {
+    @State private var text: String = ""
```

Comment:
### Informações de modelo
Acredito que vá ser necessário criar uma viewModel para a Cutscene View por conta das informações dentro dela, na hora da construção, mas não é estritamente necessário nesse momento, pode ser implementado depois conforme o app for sendo construído.